lib/mix/tasks/kitto.gen.dashboard.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Mix.Tasks.Kitto.Gen.Dashboard do
+  use Mix.Task
+  import Mix.Generator
+
+  @shortdoc "Generates a new empty dashboard template"
+
+  @template Path.join Path.expand("./templates", __DIR__), "dashboard.html.eex"
+
+  @moduledoc """
+  Generates a new empty dashboard template
+
+  Usage:
+
+      $ mix kitto.gen.dashboard all_the_data
+      # generates `dashboards/all_the_data.html.eex`
+  """
+
+  def run(argv) do
```

Comment:
Since this is documented fully in the moduledoc better add `@doc false` in all the `run` functions.